chocolatey/plugins/modules/win_chocolatey_facts.py

```diff
@@ -55,6 +71,40 @@
 - name: Displays the Outdated packages
   debug:
     var: ansible_chocolatey.outdated
+
+- name: Gather facts from chocolatey, except outdated packages
+  win_chocolatey_facts:
+    gather_subset: '!outdated'
+
+- name: Displays the collected facts from chocolatey without the outdated packages
+  debug:
+    var: ansible_chocolatey.packages
+
+- name: Clear existing facts from chocolatey
+  ansible.builtin.meta: clear_facts
+
+- name: Gather facts from chocolatey, except outdated packages
+  win_chocolatey_facts:
+    gather_subset:
+    - '!outdated'
+
+- name: Displays the collected facts from chocolatey without the outdated packages
+  debug:
+    var: ansible_chocolatey
+
+- name: Clear existing facts from chocolatey
+  ansible.builtin.meta: clear_facts
+
+- name: Gather config and feature facts only from chocolatey
+  win_chocolatey_facts:
+    gather_subset:
+    - '!all'
+    - 'config'
+    - 'feature'
```

Comment:
Hmm, starting to see a bit more of what you're doing for with these examples but tbh I think the `[!all, other]` and `[ !notthis ]` is backwards. 

I feel like it makes more sense to do `[a, b]` and `[all, !a]` rather than `[!all, a, b]` and `[!a]`.

If it's an established pattern I _guess_ we should follow suit but it's very counterintuitive and easy to mess up this way. It _says on the tin_ that we're specifying a subset to gather. A "subset" by definition means "!all" if we don't say "all" imo.

Reply:
Full commentary below (and it's changed now), but it does make sense... in a certain context (that being the sheer number of options for the main `gather_subset`... and the addition of a `!min` alongside `!all`)